[Bug] update running job, will set job clusterId to null, leading to Streampark cannot get status for the job

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-streampark/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### Java Version

1.8

### Scala Version

2.11.x

### StreamPark Version

dev

### Flink Version

1.16.2

### deploy mode

yarn-application

### What happened

deploy mode:
yarn application
yarn session

below is the  error example for yarn application mode
1. a running job
![image](https://github.com/user-attachments/assets/a7140209-4be8-4ee4-afea-0137b0ff35cd)
job cluster_id is application_1726758048768_39519
![image](https://github.com/user-attachments/assets/39a5a0ce-8024-4eaa-b5a0-4e2c0e29be96)


2. update job and submit

![image](https://github.com/user-attachments/assets/a46eedca-73d0-4fd8-ab7c-c7563aeb8655)
flink/app/update controller did not pass clusterId during update job
![image](https://github.com/user-attachments/assets/89cbf294-89a9-49bd-8347-164a65413812)

after update job， job cluster_id is set to null
![image](https://github.com/user-attachments/assets/4047d150-18a0-454b-b0e2-0ac61488dfc9)


3.  stop  the job

the job stoped failed
![image](https://github.com/user-attachments/assets/79767f9b-028e-4abe-b827-19d3b71409c1)

error stack
```
java.util.concurrent.CompletionException: java.lang.reflect.InvocationTargetException
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:273)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:280)
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1606)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.streampark.flink.client.FlinkClient$.$anonfun$proxy$1(FlinkClient.scala:89)
	at org.apache.streampark.flink.proxy.FlinkShimsProxy$.$anonfun$proxy$1(FlinkShimsProxy.scala:72)
	at org.apache.streampark.common.util.ClassLoaderUtils$.runAsClassLoader(ClassLoaderUtils.scala:44)
	at org.apache.streampark.flink.proxy.FlinkShimsProxy$.proxy(FlinkShimsProxy.scala:72)
	at org.apache.streampark.flink.client.FlinkClient$.proxy(FlinkClient.scala:84)
	at org.apache.streampark.flink.client.FlinkClient$.cancel(FlinkClient.scala:62)
	at org.apache.streampark.flink.client.FlinkClient.cancel(FlinkClient.scala)
	at org.apache.streampark.console.core.service.application.impl.ApplicationActionServiceImpl.lambda$cancel$0(ApplicationActionServiceImpl.java:314)
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1604)
	... 3 more
Caused by: java.lang.IllegalArgumentException: [StreamPark] getClusterClient error. No cluster id was specified. Please specify a cluster to which you would like to connect.
	at org.apache.streampark.common.util.AssertUtils$.required(AssertUtils.scala:59)
	at org.apache.streampark.flink.client.trait.YarnClientTrait.executeClientAction(YarnClientTrait.scala:59)
	at org.apache.streampark.flink.client.trait.YarnClientTrait.doCancel(YarnClientTrait.scala:90)
	at org.apache.streampark.flink.client.trait.YarnClientTrait.doCancel$(YarnClientTrait.scala:86)
	at org.apache.streampark.flink.client.impl.YarnApplicationClient$.doCancel(YarnApplicationClient.scala:47)
	at org.apache.streampark.flink.client.trait.FlinkClientTrait.cancel(FlinkClientTrait.scala:229)
	at org.apache.streampark.flink.client.trait.FlinkClientTrait.cancel$(FlinkClientTrait.scala:212)
	at org.apache.streampark.flink.client.impl.YarnApplicationClient$.cancel(YarnApplicationClient.scala:47)
	at org.apache.streampark.flink.client.FlinkClientEntrypoint$.cancel(FlinkClientEntrypoint.scala:48)
	at org.apache.streampark.flink.client.FlinkClientEntrypoint.cancel(FlinkClientEntrypoint.scala)
	... 16 more

```




### Error Exception

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!(您是否要贡献这个PR?)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/application/impl/ApplicationManageServiceImpl.java
+++ b/streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/application/impl/ApplicationManageServiceImpl.java
@@ -527,7 +527,6 @@ public class ApplicationManageServiceImpl extends ServiceImpl<ApplicationMapper,
         application.setDynamicProperties(appParam.getDynamicProperties());
         application.setResolveOrder(appParam.getResolveOrder());
         application.setExecutionMode(appParam.getExecutionMode());
-        application.setClusterId(appParam.getClusterId());
         application.setFlinkImage(appParam.getFlinkImage());
         application.setK8sNamespace(appParam.getK8sNamespace());
         application.updateHotParams(appParam);
@@ -559,6 +558,7 @@ public class ApplicationManageServiceImpl extends ServiceImpl<ApplicationMapper,
                 break;
             case REMOTE:
             case YARN_SESSION:
+                application.setFlinkClusterId(appParam.getFlinkClusterId());
             case KUBERNETES_NATIVE_SESSION:
                 application.setFlinkClusterId(appParam.getFlinkClusterId());
                 break;

--- a/streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/application/impl/ApplicationManageServiceImpl.java
+++ b/streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/application/impl/ApplicationManageServiceImpl.java
@@ -558,7 +558,6 @@ public class ApplicationManageServiceImpl extends ServiceImpl<ApplicationMapper,
                 break;
             case REMOTE:
             case YARN_SESSION:
-                application.setFlinkClusterId(appParam.getFlinkClusterId());
             case KUBERNETES_NATIVE_SESSION:
                 application.setFlinkClusterId(appParam.getFlinkClusterId());
                 break;

